### PR TITLE
🕸📦🔺🗓 webpack update

### PR DESCRIPTION
**Pull request by AWare**
This upgrades webpack to the latest version. It's much faster.

Also, it reworks the weekly renewal flow to not use react switch button, which raised merry hell with the build. 

image

### CODE REVIEW

**Comment by AWare** [comment]
current status: builds, need to replace renewal payment switch

**Comment by AWare** [comment]
new: 
image

old:
image

File: package.json [hunk 1]
```diff
@@ -4,20 +4,23 @@
...
-    "babel-core": "6.26.0",
-    "babel-loader": "^6.2.4",
+    "babel-core": "^6.26.0",
+    "babel-loader": "^7.1.2",
     "babel-plugin-transform-flow-strip-types": "^6.22.0",
+    "babel-plugin-transform-object-rest-spread": "^6.26.0",
     "babel-plugin-transform-runtime": "^6.23.0",
     "babel-preset-env": "^1.6.1",
-    "babel-preset-es2015": "^6.9.0",
+    "babel-preset-es2015": "^6.24.1",
```
Comment (paulbrown1982) [inline package.json]: es2015 is deprecated - https://twitter.com/housecor/status/935551724911251461

Comment (AWare) [reply package.json]: This was needed for the react-switch-button. Have removed.

**Comment by AWare** [comment]
This now brings most of our JS deps up to date and eslints our es6. Our jsx files are not, but that is worthy of a whole PR.

**Pull request merged**